libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -111,6 +111,14 @@ const AP_Param::GroupInfo AP_Arming::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("BARO2GPS",     8,     AP_Arming, _baro2gps, 0),
 
+    // @Param: ARSPD_MAX
+    // @DisplayName: Maximum valid PreArm airspeed
+    // @Description: Maximum airspeed value to pass airspeed arming check. Set 0 to disable checking.
+    // @Range: 0.1 20
```

Comment:
@peterbarker how do we make this only show in documentation for Plane and Rover? I think you've made the parser only look for tagged Values...

Reply:
`@Range{Rover}:` and `@Range{Plane}:` each on their own line.

Reply:
@WickedShell Have you actually tested that adding those entirely removes the parameter from the other vehicles? Because this:
https://github.com/ArduPilot/ardupilot/blob/4c88a1e9de4f0f610800e08e954c9592219a9bfa/Tools/autotest/param_metadata/param_parse.py#L185-L186

and this:

https://github.com/ArduPilot/ardupilot/blob/4c88a1e9de4f0f610800e08e954c9592219a9bfa/Tools/autotest/param_metadata/param_parse.py#L203-L210

are limited to checking Value.

Reply:
per-vehicle parameters are working.